src/components/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC, memo } from 'react';
+import { NavLink, Outlet } from 'react-router-dom';
+
+type Active = {
+  isActive: boolean
+};
+
+export const Header: FC = memo(() => {
+  const navLinkClass = ({ isActive }: Active) => (
+    isActive ? 'navbar-item is-tab is-active' : 'navbar-item is-tab'
+  );
```

Comment:
Better to create ActiveNavLinkComponent which return NavLink and add active class